charts/kubearchive/values.yaml

```diff
@@ -47,6 +47,21 @@ kubearchive:
           - apiVersion: v1
             kind: Event
 
+# values used to create the api-server
+apiServer:
+  name: "kubearchive-api-server"
+  # NOTE - Helm does not resolve ko path. Needs to be set on `helm install` command, e.g:
+  # helm install -n default <chart-name> charts/kubearchive \
+  # --set-string apiServer.image=$(ko build github.com/kubearchive/kubearchive/cmd/api)
+  image: ko://github.com/kubearchive/kubearchive/cmd/api
+  serviceAccount: api
+  clusterRole: api
+  clusterRoleBinding: api
```

Comment:
If you call everything using `.name`, this can be removed.

Reply:
Addressed